.github/workflows/docker-image.yml

```diff
@@ -1,14 +1,27 @@
 ---
 name: Docker Build and Push
-on:
+
+on: 
   push:
+    branches:
+      - master
+      - main
+      - test/*   # Only branches with test/ prefix will produce an image on the dockerhub account
+    paths-ignore:
+      - 'docs/**'
+      - 'requirements.docs.txt'
+      - 'mkdocs.yml'
+      - 'CNAME'
+
+  release:
+    types: [created]
+
+  pull_request:
     paths-ignore:
       - 'docs/**'
-  workflow_run:
```

Comment:
This was causing this workflow to checkout master instead of the PR we wanted to look at.

So we were pushing to master on every PR. 

We should have checked out `github.event.workflow_run.head_branch` instead